pkg/network/apis/network/validation/netid.go

```diff
@@ -0,0 +1,27 @@
+package validation
+
+// Accessor methods to annotate NetNamespace for multitenant support
+import "fmt"
```

Comment:
Shouldn't there be accompanying removal of `pkg/network/netid.go` ?

Reply:
Only validation uses this, I'd rather remove duplicates, unless they are strictly necessary. From what I've checked - they should not.